apps/chatbot/src/modules/utils.py

```diff
@@ -8,7 +8,7 @@
 
 AWS_ACCESS_KEY_ID = os.getenv("CHB_AWS_ACCESS_KEY_ID")
 AWS_SECRET_ACCESS_KEY = os.getenv("CHB_AWS_SECRET_ACCESS_KEY")
-AWS_DEFAUL_REGION = os.getenv("GOOGLE_AND_REDIS_AWS_DEFAULT_REGION")
+AWS_DEFAUL_REGION = os.getenv("CHB_GOOGLE_AWS_DEFAULT_REGION")
```

Comment:
ssm parameters are in the aws default region

```suggestion
AWS_DEFAULT_REGION = os.getenv("CHB_AWS_DEFAULT_REGION")
```